buildbot/master/files/config/steps.yml

```diff
@@ -27,6 +27,14 @@ mac-nightly:
   - ./mach package --release
   - ./etc/ci/upload_nightly.sh mac
 
+linux-rel-intermittent:
+  - ./mach build --release
+  - ./etc/ci/check_intermittents.sh
```

Comment:
You can add `--log-raw intermittents.log` so that Buildbot will pick up the log file automatically.